### PR TITLE
Adding Russian dictionary for hunspell

### PR DESCRIPTION
**Pull request by kaven96**
###### Motivation for this change
There was no Russian dictionary for hunspell. This commit adds it.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by 7c6f434c** [comment]
You are doing a PR from `master` of your own repository (a risky idea), and apparently you have force-pushed `ksh` addition over the dictionary addition

**Comment by KireevArtur** [comment]
@7c6f434c can you remove it and I will make new PR?

File: pkgs/development/libraries/hunspell/dictionaries.nix [hunk 1]
```diff
@@ -250,6 +250,41 @@ let
...
 
+  mkDictFromBB =
+    { shortName, shortDescription, dictFileName }:
+    stdenv.mkDerivation rec {
+      name = "hunspell-dict-${shortName}-BB-${version}";
+      version = "20131101";
+      folderName = "ru_RU_UTF-8_${version}";
+
+      src = fetchurl {
+        url = "https://bitbucket.org/Shaman_Alex/russian-dictionary-hunspell/downloads/ru_RU_UTF-8_${version}.zip";
```
Comment (7c6f434c) [inline pkgs/development/libraries/hunspell/dictionaries.nix]: I think if you hardcode a single-language source url, it is fine but then there is no point in defining a function.

**Comment by 7c6f434c** [comment]
Well, you can also close your own PR, or we can proceed here if in the next few days you don't plan to push any changes unrelated to the current PR into your fork's master branch

**Comment by KireevArtur** [comment]
I think, it's better to close it, because I want to clone NixOS/nixpkgs again and add my changes through branches.

**Pull request closed without merging**